.circleci/config.yml

```diff
@@ -39,11 +39,11 @@ jobs:
       - run:
           name: Install tools
           command: |
-            make -f Makefile.tools github-release
-            # this is all for package_cloud :/
-            sudo apt update -q 
-            sudo apt install -yq ruby ruby-dev build-essential
+            # this is required for the u2f to work linux & for package_cloud :/
+            sudo apt update -q
+            sudo apt install -yq ruby ruby-dev build-essential libusb-1.0.0-dev libusb-dev
```

Comment:
I think you want to install libusb* during the `dist-linux` job, and not this one.